lua/bufferline/highlight.lua

```diff
@@ -8,26 +8,42 @@ local icons = require 'bufferline.icons'
 
 -- Setup the highlight groups used by the plugin.
 hl.set_default_link('BufferCurrent', 'BufferDefaultCurrent')
+hl.set_default_link('BufferCurrentERROR', 'BufferDefaultCurrentERROR')
```

Comment:
Changes to this file allow users to customize the color of the diagnostics.

Capitals are used to make the code simpler. If we use capitals, we can do `'Buffer' .. buffer_status .. severity[s]` to generate the name of the highlight group. Otherwise, we would have to do some kind of `severity[s]:sub(1, 1) .. severity[s]:sub(2):lower()` shenanigans which would lower performance by quite a bit.